lib/NormalModule.js

```diff
@@ -370,13 +370,12 @@ class NormalModule extends Module {
 			compilation,
 			fs
 		);
-
 		runLoaders(
 			{
 				resource: this.resource,
 				loaders: this.loaders,
 				context: loaderContext,
-				readResource: fs.readFile.bind(fs)
```

Comment:
This is a breaking change to filesystem interface. We can't do that. You can't tell for sure that this is a CachedInputFileSystem from enhanced-resolve, since a plugin could override it.

I'm not willing to merge this change if this isn't a bigger performance improvement. And I doubt it is.

Reply:
I see... are you saying it would break for folks who are overriding the default `inputFileSystem`?

Reply:
@sokra if u really think this too risky, you are welcome to close pr